Automattic/wp-calypso
# Stats: Adding custom quantity option for the chart

Related to #82300

## Proposed Changes

* extend the Stats chart component to accept a custom quantity of bars

## Testing Instructions



* open the live branch and go to the Stats
* verify that the page works; you can change the period and use chat's arrows - nothing should change compared to the current state
* apply feature flag `?flags=stats/date-control`
* verify that the number of bars changes if you use new date control and select a shortcut for the `last 7 days`
* Note that this only works for `days` as a period at the moment

## Pre-merge Checklist
